octobot_trading/exchanges/traders/trader.py

```diff
@@ -671,5 +671,8 @@ def _has_open_position(self, symbol):
         :param symbol: the position symbol
         :return: True if open position for :symbol: exists
         """
-        return len(self.exchange_manager.exchange_personal_data.positions_manager.get_symbol_positions(
-            symbol=symbol)) != 0
+        return any(
```

Comment:
wouldn't this change break the `set_position_mode` `if self._has_open_position(symbol):` condition and make it impossible to set a position mode on a empty position ? If yes then I think we should adapt `set_position_mode` to work on empty positions as well (I believe we should be able to change the position mode before creating the position)

Reply:
I'm not sure if I get that right. As far as it looks to me, we can change the mode on an 0 size position, but we cant when size != 0.
I just tried it and it raises TooManyOpenPositionError when open position on the exchange and it doesnt when size is 0

Reply:
Ok then let's forget what i said, your change is right 💯